src/ripple/rpc/handlers/ServerInfo.cpp

```diff
@@ -22,13 +22,298 @@
 #include <ripple/app/reporting/P2pProxy.h>
 #include <ripple/json/json_value.h>
 #include <ripple/net/RPCErr.h>
+#include <ripple/protocol/LedgerFormats.h>
+#include <ripple/protocol/SField.h>
+#include <ripple/protocol/TER.h>
+#include <ripple/protocol/TxFormats.h>
+#include <ripple/protocol/digest.h>
 #include <ripple/protocol/jss.h>
 #include <ripple/rpc/Context.h>
 #include <ripple/rpc/Role.h>
-#include <ripple/rpc/impl/TransactionSign.h>
+#include <boost/algorithm/string.hpp>
+#include <sstream>
 
 namespace ripple {
 
+class Definitions
+{
+private:
+    std::string
+    translate(std::string inp)
+    {
```

Comment:
Separate the declarations from the implementations. We do that so it's easier to see class interfaces. (I honestly don't care that much, but it is our code style, and other people do strongly prefer it).

Reply:
Fixed in https://github.com/XRPLF/rippled/pull/4703/commits/c34bcd0a628f0c866b0b5bcef0aa5af1e22d43aa